.github/workflows/release.yml

```
@@ -0,0 +1,86 @@
+          node --check web/app.js
+      - name: Build wheel and source archive
+        run: uv build --out-dir dist
+      - name: Verify clean wheel install and runtime
+        shell: bash
+        run: |
+          wheels=(dist/*.whl)
+          test "${#wheels[@]}" -eq 1
+          python scripts/verify_package.py "${wheels[0]}"
+      - name: Create checksums
+        working-directory: dist
+        run: shasum -a 256 * > SHA256SUMS
```

actionlint (1.7.12)

[error] shellcheck reported issue in this script: SC2035:info:1:15: Use ./*glob* or -- *glob* so names with dashes won't become options

(shellcheck)